ui/pages/confirmation/templates/index.js

```diff
@@ -127,13 +127,15 @@ function getAttenuatedDispatch(dispatch) {
  * @param {object} history - The application's history object.
  * @param {Function} setInputState - A function that can be used to record the
  * state of input fields in the templated component.
+ * @param {number} unnaprovedTxsCount - The number of unapproved transactions
  */
 export function getTemplateValues(
   pendingApproval,
   t,
   dispatch,
   history,
   setInputState,
+  unnaprovedTxsCount,
```

Comment:
Same here too, so it'd be grouped in both here in the `getTemplateValues` function, but also the `getValues` implementations?

Reply:
Fixed.